members/views/departmentView.py

```diff
@@ -11,7 +11,11 @@ def departmentView(request, unique=None):
         "members/department_list.html",
         {
             "departments": filter(
-                lambda dep: dep.address.region != "", Department.get_open_departments()
+                lambda dep: dep.isVisible,
```

Comment:
Nu hvor region er blevet påkrævet kan vi vel bare sætte departments lig med `get_open_departments ` uden de har filtre

Reply:
Ja det kan vi vidst egentligt nok. Og hvad angår at sætte isVisible ind i get_open_departments, jeg var bare ikke sikker på om den funktion blev brugt andre steder hvor det måske ikke var vigtigt hvorvidt de var synlige eller ej, men bare om de var åbne.

Reply:
Tænker specifikt at den kan bruges til statistik osv, hvor man nok gerne vil vide at den eksisterer og er åben og aktiv, selvom den ikke er synlig for offentligheden.

Reply:
@Rotendahl